python/nammu/view/AtfAreaView.py

```diff
@@ -248,12 +248,12 @@ def get_viewport_carets(self):
         and will give the range from the top of the file block to the lower
         extent of the two viewports.
         '''
-        try:
+        if isinstance(self.container, JScrollPane):
             # Assumes a single edit pane
```

Comment:
Very very minor, but now that this is an `if` and not a `try`, I would change the comment to something other than "assumes".